src/hooks/useNavigation.js

```diff
@@ -19,6 +19,19 @@ export const useNavigation = defaultSection => {
 
 const handleKeyPress = (event, focusables, defaultSection) => {
   const { key } = event;
+  const { activeElement } = document;
+  let nextTabIndex;
+
+  // If element has hash then focus that element and scroll to that element
+  if(activeElement.hash) {
+    const targetElement = document.getElementById(
+      activeElement.hash.substring(1),
+    )
+
+    nextTabIndex = Array.prototype.indexOf.call(focusables, targetElement);
+
+    focus(focusables, nextTabIndex)
+  }
```

Comment:
That's cool!

Small comment:

If you join that condition to the ones that start on line 56, you won't have to call the `focus` function. You will just need to define `nextTabIndex`, and the code below will take care of the rest.